contentcuration/contentcuration/ricecooker_versions.py

```diff
@@ -1,8 +1,14 @@
 import xmlrpclib
+from socket import gaierror
 
-pypi = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
+VERSION_OK = "0.5.6"
+
+try:
+	pypi = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
+	VERSION_OK = pypi.package_releases('ricecooker')[0]
```

Comment:
Can you put in a default version for this? Other parts will expect this version to be defined.